deploy_website.sh

```diff
@@ -34,6 +34,10 @@ cd $DIR
   :okhttp-urlconnection:dokkaGfm \
   :okhttp:dokkaGfm
 
+# mkdocs doesn't correctly handle the .md paths in package-list, so post-process.
+# See https://github.com/square/okhttp/issues/7338
+find docs/4.x -name package-list -exec sed -i '' -e s/index.md$// -e s/\.md$/\\//g {} \;
```

Comment:
Looks good, but I would tweak it slightly to do:

```
find docs/4.x -name package-list -exec sed -i '' -e 's/index.md$/index.html/' -e 's/\.md$/\.html/' -e '1 s/gfm/html/' -e '2 s/md/html/' {} \;
```

That keeps the `index.html` and also updates the first two lines to remove references to `gfm` and `md`. After those changes it produces the same text as my [manually updated `package-list`](https://colinwhite.me/okhttp3-package-list).